### PR TITLE
Add 'install' make target

### PR DESCRIPTION
**Pull request by rodrigomelo9**
Hi. I recently discovered this project and added yosys-slang to one of my Dockerfiles (https://github.com/PyFPGA/containers/blob/main/recipes/synthesis). I noticed that this typical option was missing, so I added it, taking inspiration from the ghdl-yosys-plugin.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,6 +54,10 @@ clean: clean-objects
 
 clean-all: clean clean-slang
 
+install: build/slang.so
+	$(YOSYS_CONFIG) --exec mkdir -p $(DESTDIR)$(PLUGINDIR)
+	$(YOSYS_CONFIG) --exec cp $< $(DESTDIR)$(PLUGINDIR)
```
Comment (povik) [inline Makefile:59]: We are invoking yosys-config both here and in the assignment to the PLUGINDIR variable, I think one of those is superfluous

**Review by povik — COMMENTED** [review]
Thanks! I left one comment

File: Makefile (hunk 1 above)
Comment (rodrigomelo9) [reply Makefile]: Welcome :-) I just simplified what is done at `install`. Let me know what you think or what your proposal is.

**Review by povik — APPROVED** [review]
Looks fine

**Pull request merged**